Sources/Basics/Vendor/Triple+Platforms.swift

```diff
@@ -239,53 +239,6 @@ extension Triple {
     }
   }
 
-  // The Darwin platform version used for linking.
-  public var darwinLinkerPlatformVersion: Version {
```

Comment:
Removed as unused, previously it was incorrectly used as a substitute for `nil` where `Triple.Version` should've been optional in the first place.